dashboard/views.py

```diff
@@ -107,7 +92,7 @@ def file_access_within_week(request, course_id=0):
         "grade": grade,
         "course_id": course_id
     }
-    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, None, json.dumps(data))
+    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, extra=data)
```

Comment:
is there a way in the library we could mention this `extra` column data type so that we could make this column as JSON Obj instead of `longtext` now

Reply:
It doesn't look like we can support json column because mysql library we're using doesn't want to support it. https://github.com/PyMySQL/mysqlclient-python/issues/213